sample.env

```diff
@@ -1,305 +1,421 @@
+
+# If connecting to PGDB over websocket:
+INSTANCE_CONNECTION_NAME=
+
+# Python timezone for the CV Manager (You can list pytz timezones with the command 'pytz.all_timezones')
+TIMEZONE="US/Mountain"
+
+# WZDx API key and endpoint for pulling WZDx data into the CV Manager
+WZDX_API_KEY=
+WZDX_ENDPOINT=data.cotrip.org
+
+# Contact Support Menu Email Configuration
+CSM_EMAIL_TO_SEND_FROM=
+CSM_EMAIL_APP_USERNAME=
+CSM_EMAIL_APP_PASSWORD=
+CSM_EMAILS_TO_SEND_TO=
+CSM_TARGET_SMTP_SERVER_ADDRESS=
+CSM_TARGET_SMTP_SERVER_PORT=587
+CSM_TLS_ENABLED=true
+CSM_AUTH_ENABLED=true
+CSM_EMAIL_APP_USERNAME=
+CSM_EMAIL_APP_PASSWORD=
+
+
+######## -------- "webapp" Docker Profile Services
+# Run webapp service for cvmanager
+# Requires: basic
+# Compose file: docker-compose.yml
+# cvmanager_webapp
+# - React frontend for cvmanager
+
+# Mapbox token for map rendering in the webapp
+MAPBOX_TOKEN=
+
+# DOT_NAME must be set for the DOT name to correctly populate when building an image for deployment
+DOT_NAME="CDOT"
+
+# Initial map viewport
+MAPBOX_INIT_LATITUDE="39.7392"
+MAPBOX_INIT_LONGITUDE="-104.9903"
+MAPBOX_INIT_ZOOM="10"
+
+VIEWER_MSG_TYPES='BSM'
+
+CVIZ_API_SERVER_URL=http://${DOCKER_HOST_IP}:8089
+CVIZ_API_WS_URL=ws://${DOCKER_HOST_IP}:8089
+
+# Webapp themes: light, dark, cdotDark
+# base theme is used by default, dark theme is used if browser is set to dark mode
+WEBAPP_THEME_LIGHT="cdotDark" # if not set, defaults to 'light'
+WEBAPP_THEME_DARK="cdotDark" # if not set, defaults to 'dark'
+
+# Webapp logo to use, imported into docker image as volume. Set the full path to the image, for light and dark mode
+WEBAPP_LOGO_PNG_ROOT_FILE_PATH_LIGHT=./webapp/cdot_icon.png
+WEBAPP_LOGO_PNG_ROOT_FILE_PATH_DARK=./webapp/cdot_icon.png
+
+
+######## -------- "intersection" Docker Profile Services
+# Run connected intersection services
+# Requires: basic
+# Compose file: docker-compose-intersection.yml
+# Services:
+# - kafka
+#   - Message broker for communication between conflictmonitor intersection services
+# - kafka_init
+#   - Initialize kafka topics, then die
+# - intersection_api
+#   - Java backend api for intersection/conflictmonitor services
+# - mongodb_container
+#   - MongoDB database for intersection/conflictmonitor data
+
+#### ---- intersection_api
+
+# GitHub Token (Required for Intersection API) - See services/intersection-api/README.md#github-token for steps to generate
+MAVEN_GITHUB_TOKEN=
+MAVEN_GITHUB_ORG=usdot-jpo-ode
+
+#Specify MongoDB connection parameters
+DB_HOST_IP=${DOCKER_HOST_IP}
+DB_HOST_PORT=27017
+
+KAFKA_BOOTSTRAP_SERVERS=${DOCKER_HOST_IP}:9092
+KAFKA_BROKER_PORT=9092
+
+CM_MONGO_AUTH_DB=ConflictMonitor
+
+CM_SERVER_URL=http://${DOCKER_HOST_IP}:8082
+KAFKA_BROKER_IP=${DOCKER_HOST_IP}
+
+# Startup delay of intersection_api, to wait for kafka topics to be created by kafka_init
+CM_STARTUP_DELAY_SECONDS=90
+
+#### ---- mongodb_container
+
+# The Username and passwords to use for accessing mongoDB.
```

Comment:
Maybe don't capitalize username here since it doesn't really match the rest of the line? Just a nitpick though.